vllm_gaudi/v1/kv_offload/worker/cpu_hpu.py

```
@@ -0,0 +1,379 @@
+                                                     head_size=256)  #(num_blocks * block_size, num_kv_heads, head_size)
+        test_shape = (2, test_shape[0] // 128, 128, test_shape[1], test_shape[2])
+
+        has_layers_dim = False
+        split_k_and_v = False
+        if len(gpu_shape) != len(test_shape):
+            # cross-layers tensor
+            # shape is (num_blocks, ...)
+            assert len(gpu_shape) == len(test_shape) + 1
+            has_layers_dim = True
+            # prepend a dummy num_layers=80 to test_shape
+            test_shape = (80, ) + test_shape
```

Ruff (0.15.10)

[warning] Consider `(80, *test_shape)` instead of concatenation

Replace with `(80, *test_shape)`

(RUF005)